### PR TITLE
feat: airdrop confirmed transition

### PR DESCRIPTION
**Pull request by pedrorezende**
This is not the transition we've initially intended to make, but it looks ok and we can move forward with other things

### CODE REVIEW

**Comment by mateuszjasiuk** [comment]
Sneaked in one title fix ;)

**Review by mateuszjasiuk — APPROVED** [review]
LGTM! But I've also added 2 commits so it would be  nice to have another review  :)

**Comment by pedrorezende** [comment]
Request to review the latest changes, @mateuszjasiuk

File: apps/airdrop/src/App/AnotherWays.tsx [hunk 1]
```diff
@@ -24,6 +25,12 @@ export const AnotherWays: React.FC<AnotherWaysProps> = (props) => {
   const metamask = (window as MetamaskWindow)?.ethereum;
   const columns = !metamask && !keplr ? "1fr 1fr" : "1fr 1fr 1fr";
 
+  useEffect(() => {
+    if (typeof props.reset === "boolean") {
```
Comment (emccorson) [inline apps/airdrop/src/App/AnotherWays.tsx:29]: Just to check I'm understanding this: The `reset` prop is not really a boolean; it's like a signal which every time it gets changed will tell the component to uncheck the TOS checkbox?
That's pretty cool! I had to read it a few times to get my head around it though :smile:

**Review by emccorson — APPROVED** [review]
Code looks good to me and the animation is nice :+1:

**Review by mateuszjasiuk — APPROVED** [review]

**Review by mateuszjasiuk — APPROVED** [review]

**Pull request merged**